src/GlobalDiffEq.jl

```diff
@@ -25,6 +27,71 @@ function DiffEqBase.__solve(prob::Union{DiffEqBase.AbstractODEProblem,DiffEqBase
     return sol
 end
 
-export GlobalRichardson
+struct GlobalAdjoint{A} <: GlobalDiffEqAlgorithm
+    alg::A
+end
+
+function DiffEqBase.__solve(prob::Union{DiffEqBase.AbstractODEProblem,DiffEqBase.AbstractDAEProblem},
+                            alg::GlobalAdjoint, args...;
+                            dt,kwargs...)
+
+    opt = Dict(kwargs)
+    otheropts = delete!(copy(opt), :dt)
+    tstops = get(opt, :tstops, range(prob.tspan[1], stop=prob.tspan[2], step=dt))
+    local sol
+
+    n = length(prob.u0)
+    k = 2 # 2 random vectors
+    C = 11/6 # for k = 2
+    E₂ = 2/π
+    Eₙ = vec_expt_full(n)
+    gtol_get = get(opt, :abstol, 1e-6) # target gtol is abstol? abstol cannot be zero?
```

Comment:
we should generalize it to relative and abstol, but start with just abstol I guess